siphon/acis.py

```diff
@@ -0,0 +1,51 @@
+import json
+import urllib.request
+import urllib.parse
+import json
+import socket
+
+def acisRequest(method, params):
+    """
+    
+    This function will make a request to the ACIS Web Services API for data
+    based on the given method (StnMeta,StnData,MultiStnData,GridData,General)
+    and parameters string. Information about the parameters can be obtained at:
+    http://www.rcc-acis.org/docs_webservices.html
+
+    If a connection to the API fails, then it returns False. A failed request
+    will have an error text in the return data produced by the remote API.
+
+    Parameters
+    ----------
+    method - The Web Services request method (StnMeta, StnData, MultiStnData, etc)
+    params - A dictionary of parameters (see API documentation)
+
+    Returns
+    ----------
+    Connection Success: A dictionary that is derived from JSON data from the remote API
```

Comment:
Would need to be updated based on discussion below. Also Sphinx will pitch a fit unless the length of "---------" matches the length of "Returns".